.github/workflows/phpunit.yml

```
@@ -17,7 +17,7 @@ jobs:
         os: [ubuntu-18.04]
         epubcheck: [4.2.4]
         prince: [12.5-1]
-        wordpress: [5.6.2]
+        wordpress: [5.6.2, 5.7]
 
     name: PHPUnit - ${{ matrix.php }}
```

For clarity, it would be good to show the WordPress version here too so that the jobs are distinguishable in the checks list, e.g.:

```suggestion
    name: PHPUnit - ${{ matrix.php }} - ${{ matrix.wordpress }}
```